src/rules/memberOrderingRule.ts

```diff
@@ -17,16 +17,55 @@
 import * as Lint from "../lint";
 import * as ts from "typescript";
 
+/* start old options */
 const OPTION_VARIABLES_BEFORE_FUNCTIONS = "variables-before-functions";
 const OPTION_STATIC_BEFORE_INSTANCE = "static-before-instance";
 const OPTION_PUBLIC_BEFORE_PRIVATE = "public-before-private";
+/* end old options */
+
+/* start new options */
+const OPTION_ORDER = "order";
+const PRESET_ORDERS: { [preset: string]: string[] } = {
```

Comment:
@lowkay @thasner @alan-agius4 (and others) would you use one of these presets? If not, what preset would you like?